aten/src/ATen/native/cuda/Normalization.cu

```diff
@@ -78,36 +78,36 @@ struct Float2 {
 
 template <typename scalar_t, typename accscalar_t>
 struct SumOp {
-  __device__ SumOp(const PackedTensorAccessor<scalar_t, 3, at::RestrictPtrTraits>& t) : tensor(t) {}
+  __device__ SumOp(const PackedTensorAccessor<scalar_t, 3>& t) : tensor(t) {}
```

Comment:
This might be a dumb question, considering my limited knowledge of `__restrict__`. Does `__restrict__` do anything here for this unary op?

Reply:
This was just so I could pass around the tensoraccessor. :)